.gitignore

```diff
@@ -157,4 +157,4 @@ dmypy.json
 cython_debug/
 
 # Model files
-models/
+/models/
```

Comment:
wow, this is a great catch!

Reply:
Previously we didn't have a `models` dir on lower levels so it could be avoided.